### PR TITLE
Move around node content & reduce duplicate content 

### PR DESCRIPTION
**Pull request by tas50**
- Simplify the node description
- Remove the empty run list troubleshooting stuff which is in the debug doc and is better suited for that doc
- Make node object front and center and then put run list and attributes below that since those are part of the node object
- Remove all the detailed attributes stuff and instead link to the main attributes doc for additional information

### CODE REVIEW

File: content/nodes.md [hunk 1]
```diff
@@ -118,73 +79,27 @@ copy on the Chef Infra Server at the end of each Chef Infra Client run.
...
-## Blocklist Attributes
+{{% node_run_list_format %}}
 
-{{% node_attribute_blocklist %}}
+## Managing Nodes
 
-### Allowlist Attributes
+There are several ways to manage nodes directly: via knife, Chef
+Automate, or by using command-line tools that are specific to Chef Infra
+Client.
```
Comment (IanMadd) [inline content/nodes.md]:
```suggestion
You can manage nodes directly using Knife, Chef Automate, or by using command-line tools that are specific to Chef Infra Client.
```

File: content/nodes.md [hunk 2]
```diff
@@ -118,73 +79,27 @@ copy on the Chef Infra Server at the end of each Chef Infra Client run.
...
 
-{{% node_attribute_allowlist %}}
+- [knife](/workstation/knife/) can be used to create, edit, view, list, tag, and delete nodes.
```
Comment (IanMadd) [inline content/nodes.md]:
```suggestion
- [Knife](/workstation/knife/) can be used to create, edit, view, list, tag, and delete nodes.
```

File: content/nodes.md [hunk 3]
```diff
@@ -118,73 +79,27 @@ copy on the Chef Infra Server at the end of each Chef Infra Client run.
...
+- knife plug-ins can be used to create, edit, and manage nodes that are located on cloud providers.
```
Comment (IanMadd) [inline content/nodes.md]:
```suggestion
- Knife plug-ins can be used to create, edit, and manage nodes that are located on cloud providers.
```

**Review by kagarmoe — APPROVED** [review]
tenor-175332020

**Pull request merged**